app/mailers/user_mailer.rb

```diff
@@ -353,11 +353,11 @@ def prompter_notification(work_id, collection_id=nil)
   def delete_work_notification(user, work)
     @user = user
     @work = work
-    work_copy = generate_attachment_content_from_work(work)
-    work_copy = ::Mail::Encodings::Base64.encode(work_copy)
```

Comment:
I think we would still want base64 encoding (plus the `encoding` option on `attachments`) to avoid long lines (https://otwarchive.atlassian.net/browse/AO3-5453).

Reply:
We should keep the calls `::Mail::Encodings::Base64.encode` on the final content strings as well.